packages/dependency-extraction-webpack-plugin/test/fixtures/combine-assets/webpack.config.js

```diff
@@ -12,7 +12,9 @@ module.exports = {
 		new DependencyExtractionWebpackPlugin( {
 			combineAssets: true,
 			requestToExternalModule( request ) {
-				return request.startsWith( '@wordpress/' );
+				return (
+					request.startsWith( '@wordpress/' ) || request === 'lodash'
```

Comment:
In the module tests `lodash` was not externalized but bundled. Now it is externalized (to prevent errors). This is reflected in the snapshots.

Reply:
I noticed in the original PR that `lodash` gets bundled into the fixture's build output, but I thought it's desired 🙂 Also, the tests don't look at the build output at all, only at the dependency metadata.